exercises/high-scores/.meta/HINTS.md

```diff
@@ -0,0 +1,17 @@
+### Python and Object Oriented Programming (OOP)
+
+
+
+High Scores requires that you make a **class** in Python, so you need to be familiar with the basics of  **Object Oriented Programming**.  _Don't worry, it's not as complicated as you think it is!_   Here are some tutorials and links to get you up to speed:
+
+
+
+1)   [**Object-Oriented Programming (OOP) in Python 3**](https://realpython.com/python3-object-oriented-programming/) from the Real Python website.  Goes over all the definitions and mechanics step by step.  Great for the absolute first-timer.
```

Comment:
Any strong reasoning on ordered vs unordered list for this?

Reply:
My original thought was to give a beginner (explains OOP lingo) -->intermediate (uses OOP lingo)-->advanced (talks about language internals) progression.  

 After talking with @F3PiX and others, I realize that's overkill.  Edited to have the docs link, link to class creation, and a link to data structures docs.